unblob/extractor.py

```diff
@@ -47,6 +47,12 @@ def is_recursive_link(path: Path) -> bool:
     return False
 
 
+def fix_symlink(path: Path, outdir: Path, task_result: TaskResult) -> Path:
+    # This is a temporary function for existing unit tests in tests/test_extractor.py
+    fix_extracted_directory(outdir, task_result)
+    return path
+
+
```

Comment:
I suppose it's a temporary fix and the commit message should say so.

Reply:
answered as main comment.
these changes are to be dropped.